docs/tutorials/p2p.md

```diff
@@ -102,7 +102,7 @@ Inside that handshake there was a call to the ***_read_P2P_Packet()*** function
 		if PACKET_SIZE > 0:
 			var PACKET: Dictionary = Steam.readP2PPacket(PACKET_SIZE, 0)
 
-			if PACKET.empty() or PACKET == null:
+			if PACKET.is_empty() or PACKET == null:
```

Comment:
This is the right change in the wrong place.  This change should be in the next codeblock down.